### PR TITLE
AzureRT S54 PR #1 - Compute Test Fix

### PR DESCRIPTION
**Pull request by huangpf**
@hyonholee

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@huangpf__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (phuang). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by huangpf** [comment]
retest this please

**Comment by huangpf** [comment]
retest this please

File: tools/autorest.gen.cmd [hunk 1]
```diff
@@ -8,6 +8,7 @@ set source=-Source https://www.myget.org/F/autorest/api/v2
 
 set repoRoot=%~dp0..
 set autoRestExe=%repoRoot%\packages\autorest.%autoRestVersion%\tools\AutoRest.exe
+REM set autoRestExe=%repoRoot%\..\autorest\binaries\net45\AutoRest.exe
```
Comment (stankovski) [inline tools/autorest.gen.cmd]: Revert

**Pull request merged**